test/display_test.py

```diff
@@ -314,23 +314,19 @@ def test_quit__multiple(self):
 
         self.assertFalse(display.get_init())
 
-    def todo_test_set_gamma(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.display.set_gamma:
-
-        # pygame.display.set_gamma(red, green=None, blue=None): return bool
-        # change the hardware gamma ramps
-        #
-        # Set the red, green, and blue gamma values on the display hardware.
-        # If the green and blue arguments are not passed, they will both be
-        # the same as red. Not all systems and hardware support gamma ramps,
-        # if the function succeeds it will return True.
-        #
-        # A gamma value of 1.0 creates a linear color table. Lower values will
-        # darken the display and higher values will brighten.
-        #
-
-        self.fail()
+    @unittest.skipIf(
+    os.environ.get("SDL_VIDEODRIVER") == "dummy",
+    'OpenGL requires a non-"dummy" SDL_VIDEODRIVER',
```

Comment:
```suggestion
    @unittest.skipIf(
    os.environ.get("SDL_VIDEODRIVER") == "dummy" and not SDL2,
    'Can't set gamma on SDL 1 with the dummy video driver',
```

Think this is a skip that will still let us run the test on most of the Continuous Integration builds.